kotlin-checks-test-sources/src/main/kotlin/checks/MobileDatabaseEncryptionKeysCheckSample.kt

```diff
@@ -0,0 +1,91 @@
+package checks
+
+import io.realm.RealmConfiguration
+import net.sqlcipher.database.SQLiteDatabase
+
+class MobileDatabaseEncryptionKeysCheckSample(val k: Key) {
+
+    fun realm(s: String) {
+        val stringKey = "d2345678e012345678901c3456789012a456789012n45678901234o678901227" // Secondary location here
```

Comment:
How about verifying that the secondary location is really marked? You could also do this by constructing a smaller example in a new method to avoid having a `// Noncompliant@+veryLargeNumber` comment.